src/renderer/contexts/DependencyContext.tsx

```diff
@@ -91,86 +121,141 @@ const Package = memo(
+                                    </Button>
+                                </HStack>
+                            )}
+                            <AccordionButton
+                                cursor="pointer"
+                                w={4}
                             >
-                                Uninstall
-                            </Button>
+                                <Center
+                                    cursor="pointer"
+                                    w="full"
+                                >
+                                    <AccordionIcon />
+                                </Center>
+                            </AccordionButton>
                         </HStack>
-                    ) : (
-                        <Button
-                            colorScheme="blue"
-                            disabled={isRunningShell}
-                            isLoading={isRunningShell}
-                            leftIcon={<DownloadIcon />}
-                            size="sm"
-                            onClick={onInstall}
-                        >
-                            Install ({formatSizeEstimate(dep.packages)})
-                        </Button>
-                    )}
-                </Flex>
-                {progress !== undefined && (
-                    <Center
-                        h={8}
+                        {progress !== undefined && (
+                            <Center
+                                cursor="pointer"
+                                h={8}
+                                w="full"
+                            >
+                                <Progress
+                                    hasStripe
+                                    isAnimated
+                                    cursor="pointer"
+                                    value={progress}
+                                    w="full"
+                                />
+                            </Center>
+                        )}
+                    </VStack>
+                </h2>
+                <AccordionPanel pb={4}>
+                    <VStack
+                        key={dep.name}
                         w="full"
                     >
-                        <Progress
-                            hasStripe
-                            isAnimated
-                            value={progress}
-                            w="full"
-                        />
-                    </Center>
-                )}
-            </VStack>
+                        {dep.packages.map((p) => (
+                            <IndividualDep
+                                installed={!!pipList[p.packageName]}
+                                key={p.packageName}
+                                outdated={
+                                    !!pipList[p.packageName] &&
+                                    checkSemverGt(p.version, pipList[p.packageName]!)
+                                }
```

Comment:
I feel like it would be easier to just pass in `installedVersion: string | undefined` and then do these checks inside `IndividualDep`